backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -27,20 +27,29 @@ import org.apache.spark.sql.catalyst.expressions.aggregate.{AggregateExpression,
 import org.apache.spark.sql.catalyst.plans.JoinType
 import org.apache.spark.sql.execution.SparkPlan
 import org.apache.spark.sql.execution.aggregate.HashAggregateExec
+import org.apache.spark.sql.execution.command.CreateDataSourceTableAsSelectCommand
+import org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand
 import org.apache.spark.sql.expression.UDFResolver
 import org.apache.spark.sql.types._
 
 import scala.util.control.Breaks.breakable
 
 class VeloxBackend extends Backend {
-  override def name(): String = GlutenConfig.GLUTEN_VELOX_BACKEND
-  override def iteratorApi(): IteratorApi = new IteratorHandler
-  override def sparkPlanExecApi(): SparkPlanExecApi = new SparkPlanExecHandler
-  override def transformerApi(): TransformerApi = new TransformerHandler
-  override def validatorApi(): ValidatorApi = new Validator
-  override def metricsApi(): MetricsApi = new MetricsHandler
+  override def name(): String = VeloxBackend.BACKEND_NAME
+  override def buildInfo(): GlutenPlugin.BackendBuildInfo =
+    GlutenPlugin.BackendBuildInfo("Velox", VELOX_BRANCH, VELOX_REVISION, VELOX_REVISION_TIME)
```

Comment:
Can we just use `BACKEND_NAME` to replace `Velox` for consistency?

Reply:
To me the name in BackendBuildInfo is just a declarative text than BACKEND_NAME which is used for lib loading. I think it's OK to use "Velox" "VELOX" or whatever letting user know it's the Velox backend here. So using `BACKEND_NAME` will not be the best choice in this case, from my personal understanding.

Reply:
How about adding some comments for `BackendBuildInfo` to clarify it is only for declarative purpose, and should not be used in real code?

Reply:
> How about adding some comments for BackendBuildInfo to clarify it is only for declarative purpose, and should not be used in real code?

Agreed but do you think we can do that in another PR? Since it's a little bit off-topic.

Reply:
With the assumption that this PR just some code movement across modules. It's far beyond its mission to improve all relevant code's quality. ; )

Reply:
OK, makes sense.